scripts/infinity_grid.py

```diff
@@ -80,14 +80,17 @@ def apply_restore_faces(p, v):
     if restorer is not None:
         opts.face_restoration_model = restorer
 
-def prompt_replace_parse_list(in_list):
+def prompt_replace_parse_list(in_list, type):
+    if type == "pos":
+        pr = "promptreplace"
+    else: pr = "negativepromptreplace"
```

Comment:
the input is ... a string that just gets another string? why not just input the correct param name string in the first place?